Dockerfile

```diff
@@ -12,22 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.12.6
-WORKDIR /cluster-api-provider-docker
-ADD go.mod .
-ADD go.sum .
-RUN go mod download
-RUN  curl -L https://dl.k8s.io/v1.14.3/kubernetes-client-linux-amd64.tar.gz | tar xvz
-ADD cmd cmd
-ADD actuators actuators
-ADD kind kind
-ADD third_party third_party
-
-RUN go install -v ./cmd/capd-manager
+FROM golang:1.12.7
+WORKDIR /tmp
+RUN curl -L https://dl.k8s.io/v1.14.4/kubernetes-client-linux-amd64.tar.gz | tar xvz
+RUN mv /tmp/kubernetes/client/bin/kubectl /usr/local/bin
 RUN curl https://get.docker.com | sh
+COPY capd-manager /usr/local/bin
```

Comment:
This assumes `capd-manager` will always be present in `$(PWD)` and will also be the latest build. Which might not be true. So, suggest building `capd-manager`.

Reply:
This is now only really for use for go releaser. Maybe in a followup PR we should add a Dockerfile that builds one without goreleaser? I'll file an issue

Reply:
filed a general issue for improving dev builds. #98